cpp/src_prims/sparse/csr.h

```diff
@@ -675,6 +675,11 @@ class WeakCCState {
         std::cout << "Exception freeing memory for WeakCCState: " << e.what()
                   << std::endl;
       }
+
+      xa = nullptr;
```

Comment:
do we need explicitly reset these pointers, even inside the destructor?

Reply:
Nope. Another debugging artifact.